nabweb/templates/nabweb/system-info/index.html

```diff
@@ -8,7 +8,7 @@
       <div class="card">
         <div class="card-header">
           {% if os.variant == "Raspberry Pi" %}
-          <h5 class="card-title"><a class="help-link" href='https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os-32-bit' target='_blank' rel='noopener noreferrer'>{% blocktrans with variant=os.variant %}{{ variant }} OS{% endblocktrans %}</a></h5>
+          <h5 class="card-title"><a class="help-link" href='https://www.raspberrypi.org/software/operating-systems/#raspberry-pi-os' target='_blank' rel='noopener noreferrer'>{% blocktrans with variant=os.variant %}{{ variant }} OS{% endblocktrans %}</a></h5>
```

Comment:
This URI redirects to raspberrypi.com.

Reply:
It does indeed, as it did before, probably for quite some time.
Do you want to fix the URI so that it points directly to raspberry.com?

Reply:
You did fix the other URIs, so this one should probably be fixed as well ;-)

Reply:
Indeed, will do.
The subtle différence is that the other URIs could only work as raspberrypi.com based URIs, to point to the correct/current tag on the new documentation pages.